packages/redux-simple-auth/src/middleware.js

```diff
@@ -120,6 +126,35 @@ export default (config = {}) => {
             return response
           })
         }
+        case INVALIDATE_SESSION: {
+          const state = getState()
+          const authenticatorName = getAuthenticator(state)
+
+          if (authenticatorName) {
+            const authenticator = findAuthenticator(authenticatorName)
+
+            if (!authenticator) {
+              throw new Error(
+                `No authenticator with name \`${action.meta.authenticator}\` ` +
+                  'was found. Be sure you have defined it in the authenticators ' +
+                  'config.'
+              )
+            }
+
+            if (authenticator) {
```

Comment:
We shouldn't need this here since you've already checked for the presence of it above and `throw`n if it doesn't exist. Go ahead and remove this.

Reply:
👍